std/json.d

```diff
@@ -96,11 +96,34 @@ enum JSON_TYPE : byte
     FALSE    /// ditto
 }
 
+private struct DefaultPolicy { }
+
 /**
 JSON value node
 */
-struct JSONValue
+alias JSONValue = JSONValueTemplate!DefaultPolicy;
+
+/**
+A JSONValue template. Features can be enabled/disabled via setting members in $(D Policy).
+Example:
+------------
+struct MyPolicy
+{
+    enum sortObjectMembers = false;
+    alias MyObjectType(K,V) = OrderedAA!(K,V);
+}
+alias MyJSONValue = JSONValueTemplate!MyPolicy;
+------------
```

Comment:
The example here really should be in a ddoc'd unittest.  And it should show how to actually use `JSONValueTemplate`, not just in how to instantiate it, but how to use it with the rest of the module, e.g., to configure it with a different `JSONObject` type, etc..

Reply:
Agreed.  Was going to wait to fully document  until a general "approved" consensus was reached.